src/test/java/codeu/controller/ActivityFeedServletTest.java

```diff
@@ -0,0 +1,5 @@
+package codeu.controller;
+
+public class ActivityFeedServletTest {
```

Comment:
Ideally you'd write some tests.  You could skip them if you're eager to get to the open project.  But, alternatively, after you merge this PR, you could write some tests in a second PR before moving on to the open project.  I leave that up to you.